yourgov/pages/app/index.tsx

```diff
@@ -107,7 +107,7 @@ const Page: NextPageWithLayout = () => {
 												<Tag>{linkedBusinesses.abn}</Tag>
 												<Text>ABN</Text>
 											</Flex>
-											<Heading as="h2" type="h3">
+											<H3>
```

Comment:
Fixing semantic heading structure of these cards. Didn't keep `<Heading>` to be consistent with every other Card H3 in yourgov